src/components/formFields/EmailField.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { ErrorIcon } from '../Notifications/ErrorIcon';
+
+type Props = {
+  email: string,
+  isEmailEmptyError: boolean,
+  isEmailValid: boolean,
+  setEmail: React.Dispatch<React.SetStateAction<string>>
+  setIsEmailEmptyError: React.Dispatch<React.SetStateAction<boolean>>,
+  setIsEmailValid: React.Dispatch<React.SetStateAction<boolean>>,
+};
+
+export const EmailField: FC<Props> = ({
+  email,
+  isEmailEmptyError,
+  isEmailValid,
+  setEmail,
+  setIsEmailEmptyError,
+  setIsEmailValid,
+}) => {
+  const onEmailChangeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364